src/language/json/monaco.contribution.ts

```diff
@@ -4,7 +4,8 @@
  *--------------------------------------------------------------------------------------------*/
 
 import * as mode from './jsonMode';
-import { Emitter, IEvent, languages } from '../../fillers/monaco-editor-core';
+import { Emitter, IEvent, languages, Uri } from 'monaco-editor-core';
+import * as jsonService from 'vscode-json-languageservice';
```

Comment:
The types that are used from `jsonService` should be copied to `json/monaco.contribution.ts` so that the d.ts file is self-contained.

Reply:
Got it, lemme fix it.

Reply:
Though, I wonder then why we have pretty much the same setup in https://github.com/microsoft/monaco-editor/blob/main/src/language/typescript/monaco.contribution.ts#L6 :)

Reply:
Ah, sorry, it's `jsonMode` not `jsonService`.

Reply:
@aeschli looks crazy, but I'm done, sir :)